spec.html

```diff
@@ -6888,6 +6909,32 @@ <h1>
           1. Set _k_ to _k_ + 1.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+      <h1>
+        SetterThatIgnoresPrototypeProperties (
```

Comment:
does this AO belong in this PR?

Reply:
Well, it's used in this PR, and we generally don't land AOs that have no users.

Reply:
ah, sorry, i missed the usage and apparently incorrectly recalled that we'd decided on a different approach than this.